tests/.snapshots/success.txt

```diff
@@ -348,6 +348,9 @@
   ✓ it throws exception when `process isolation` is true
   ✓ it do not throws exception when `process isolation` is false
 
+   PASS  Tests\Unit\Console\Help
+  ✓ it outputs the help information when --help is used
```

Comment:
We may need an integration test on this. Do you mind of adding it?

Reply:
Sure. :+1: So just call `pest --help` and check it's in the output?

Reply:
Is that ok? Or should they be in the same file? 🤔